domino.go

```diff
@@ -283,6 +317,17 @@ type KeyValue struct {
 /************************************** GetItem ****************************************/
 /***************************************************************************************/
 type get dynamodb.GetItemInput
+type output struct {
+	*dynamodb.GetItemOutput
+	Error error
+}
+
+func (o *output) Result(item interface{}) (err error) {
```

Comment:
👍 that's what I imagined as well - this is definitely a good API to adopt moving forward